packages/react/src/components/Tag/Tag-story.js

```diff
@@ -69,6 +69,23 @@ storiesOf('Tag', module)
       },
     }
   )
+  .add(
+    'Short',
+    () => (
+      <Tag className="some-class" {...props.filter()} filter short>
```

Comment:
going off of @asudoh's suggested changes in `Tag.js`

```suggestion
      <Tag className="some-class" {...props.filter()} filter size="short">
```